### PR TITLE
S2U-17 T&Q: Question pool statistics and tags - MASTER

### PR DESCRIPTION
**Pull request by stetsche**
https://sakaiproject.atlassian.net/browse/S2U-17

### CODE REVIEW

File: samigo/samigo-app/src/webapp/jsf/questionpool/statisticsModal.jsp [hunk 1]
```diff
@@ -0,0 +1,86 @@
+<script src="/samigo-app/js/questionPoolStatistics.js"></script>
+<div id="stat-modal" class="modal fade" role="dialog" tabindex="-1">
```
[the remark this answers is not in the document]
Comment (stetsche) [reply samigo/samigo-app/src/webapp/jsf/questionpool/statisticsModal.jsp:2]: Makes sense!

**Comment by mpellicer** [comment]
Maven build fails

**Comment by mpellicer** [comment]
Sure, re-running

**Comment by stetsche** [comment]
Locally that test is passing for me.

**Pull request merged**